cryptoapis/README.md

```diff
@@ -67,36 +87,6 @@ The adapter takes the following environment variables:
 }
 ```
 
-### Height
-
-- `blockchain` or `coin`: The blockchain name (required).
-- `network`: The blockchain network name. Default: `mainnet`
-
-### Output
-
-```json
-{
-  "jobRunID": "1",
-  "data": {
-    "payload": {
-      "difficulty": 20607418304385.63,
-      "headers": 666185,
-      "chain": "main",
-      "chainWork": "000000000000000000000000000000000000000018255ab714d1a15ffccd987e",
-      "mediantime": 1610721116,
-      "blocks": 666185,
-      "bestBlockHash": "0000000000000000000cc82b0a9a6e290cd13721a1abf88fdebb37fdc927308e",
-      "currency": "BTC",
-      "transactions": 606560353,
-      "verificationProgress": 0.9999935897991173
-    },
-    "result": 666185
-  },
-  "result": 666185,
-  "statusCode": 200
-}
-```
-
 ### Balance
```

Comment:
Left this as it is @justinkaseman, because probably it could be batch-updated along other balance endpoint docs.